docs/dns-challenge/hetzner.md

```diff
@@ -0,0 +1,29 @@
+# Variables for hetzner dns-challenge
+
+| Variable       | Required | Default | Description
+|----------------|----------|---------|------------
+| dns_auth_token | yes      |         | Access token for hetzner DNS API
```

Comment:
Before we release 1.0, we should rename this variable to `dns_hetzner_auth_token.`

Reply:
You are right. Will Update

Reply:
did this in another pull request: https://github.com/T-Systems-MMS/ansible-collection-letsencrypt/pull/38